roles/openshift-applier/tasks/process-content.yml

```diff
@@ -9,9 +9,9 @@
   block:
   - fail:
       msg: "WARNING: This functionality is being deprecated. Please use the 'content.file' variable instead. See role docs for more info."
-      ignore_errors: True
```

Comment:
@etsauer I believe it's in the right place, it just needs to be moved back out to align with the `fail` task. I.e.: we want to ignore errors on the `fail` task, but not on the `command` task. Your propose change will ignore errors on the entire block